webapp/portlet/src/main/webapp/space-settings/components/SpaceSettingAddApplicationDrawer.vue

```diff
@@ -60,8 +60,8 @@ export default {
         this.applications = data.map(app => ({
           applicationName: app.applicationName,
           contentId: app.contentId,
-          description: this.$t(`SpaceSettings.application.${/\s/.test(app.displayName) ? app.displayName.replace(/ /g,'.').toLowerCase() : app.displayName.toLowerCase()}.description`),
-          displayName: this.$t(`SpaceSettings.application.${/\s/.test(app.displayName) ? app.displayName.replace(/ /g,'.').toLowerCase() : app.displayName.toLowerCase()}.title`),
```

Comment:
How will those applications names and descriptions will be translated ?

Reply:
This two lines are the cause of the problem of displaying the name and description. But right, i will try to find a solution to recover the two options.